inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -13,7 +13,7 @@
 
 
 def application_add(args: argparse.Namespace) -> str:
-    if (args.gpgKeyUri and args.gpgKeyName is None) or (args.gpgKeyName and args.gpgKeyUri is None):
+    if bool(args.gpgKeyUri) != bool(args.gpgKeyName):
         raise InbcException(
             "Source requires either both gpgKeyUri and gpgKeyName to be provided, or neither of them.")
```

Comment:
**Suggestion:** Use a more Pythonic way to check if both 'args.gpgKeyUri' and 'args.gpgKeyName' are provided or neither, by using 'all' or 'any' with a generator expression. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    if not all([args.gpgKeyUri, args.gpgKeyName]) and not all([not args.gpgKeyUri, not args.gpgKeyName]):
        raise InbcException(
            "Source requires either both gpgKeyUri and gpgKeyName to be provided, or neither of them.")
```